hudi-flink-datasource/hudi-flink/src/main/java/org/apache/hudi/table/catalog/HoodieHiveCatalog.java

```diff
@@ -475,6 +478,8 @@ public void createTable(ObjectPath tablePath, CatalogBaseTable table, boolean ig
       if (!ignoreIfExists) {
         throw new TableAlreadyExistException(getName(), tablePath, e);
       }
+    } catch (HoodieValidationException e) {
+      throw e;
```

Comment:
This catch is meaningless, we can fetch the cause of the wrapper `HoodieCatalogException` if you want to check it in the test.